light-macros/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+use anchor_syn::AccountsStruct;
+use proc_macro::TokenStream;
+use syn::{parse_macro_input, ItemStruct};
+
+mod expand;
```

Comment:
where does this come from I don't see a file or import.

Can you explain where and how the accounts are defined?

Reply:
Ah I saw it in the your other links. But it's missing in this pr right?

Reply:
Yes, I didn't push it, sorry.